setup/docker.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Coder for Docker"
+description: Learn how to run Coder with Docker.
+---
+
+Coder for Docker allows you to deploy Coder to any machine on which Docker runs
+quickly.
+
+## Prerequisites
+
+You must be using a machine that is running **macOS** and has
+[Docker Desktop](https://www.docker.com/products/docker-desktop) installed.
+
+## Installing Coder for Docker
+
+1. Launch Docker Desktop.
+
+1. If you've previously installed Coder, run `rm -rf ~/.coder` in the terminal.
+
+1. In the terminal, run the following to download the resources you need,
+   include the images, and set up your Coder deployment:
+
+   ```console
+   docker run -it --rm -p 7080:7080 -v ~/.coder:/var/run/coder -v /var/run/docker.sock:/var/run/docker.sock -e ENVBOX_IMAGE=kylecarbs/test-image-2:latest kylecarbs/test-image:0.4.0
```

Comment:
@kylecarbs could you please provide the updated command when you have it? thanks!

Reply:
It'll be:
```
docker run --rm -it -p 7080:7080 -v /var/run/docker.sock:/var/run/docker.sock -v ~/.coder:/var/run/coder codercom/coder
```
It's not live yet, but I'll put it up later today.